src/ActionsLinux.cpp

```diff
@@ -17,6 +17,30 @@
 #include <sys/ioctl.h>
 #endif
 
+char* ReadSymbolicLink(const char* path, MemAllocLinear* allocator, struct stat* stat_info)
+{
+    struct stat s;
+    if (stat_info == nullptr)
+    {
+        stat_info = &s;
+        if (lstat(path, &s) != 0)
+        {
+            Log(kError, "Failed to stat %s", path);
+            return nullptr;
+        }
+    }
+
+    const size_t bufferSize = stat_info->st_size + 1;
+    char* link_target = static_cast<char*>(LinearAllocate(allocator, bufferSize, 1));
+
+    int link_size = readlink(path, link_target, bufferSize);
+    if (link_size >= bufferSize)
+        return nullptr;
```

Comment:
Is it worth logging an error or a warning here?

Reply:
Both of the places this is called, if `nullptr` is returned then we will fail the whole action with an appropriate message. I think logging here would be overkill.